sync/evmdriver.go

```diff
@@ -84,18 +87,20 @@ reset:
 	cancellableCtx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	log.Info("Starting sync. syncID: ", syncID, " lastProcessedBlock", lastProcessedBlock)
 	// start downloading
 	downloadCh := make(chan EVMBlock, d.downloadBufferSize)
-	go d.downloader.Download(cancellableCtx, lastProcessedBlock, downloadCh)
+	go d.downloader.Download(cancellableCtx, lastProcessedBlock+1, downloadCh)
 
 	for {
 		select {
 		case b := <-downloadCh:
-			d.log.Debug("handleNewBlock")
+			d.log.Debug("handleNewBlock: ", b.Num, b.Hash, " syncID ", syncID)
 			d.handleNewBlock(ctx, b)
+			d.log.Debug("handleNewBlock done: ", b.Num, b.Hash, " syncID ", syncID)
 		case firstReorgedBlock := <-d.reorgSub.ReorgedBlock:
-			d.log.Debug("handleReorg")
-			d.handleReorg(ctx, cancel, downloadCh, firstReorgedBlock)
+			d.log.Debug("handleReorg: ", firstReorgedBlock, " syncID ", syncID)
```

Comment:
Correct me if I'm wrong, but shouldn't there be an odd number of parameters?

```suggestion
			d.log.Debug("handleReorg: ", "first reorged block", firstReorgedBlock, "syncID", syncID)
```

Reply:
No, it is arbitrary.